### ISSUE TITLE
webrtc connections fail after a BBB conference is running for about 6 hours.

### ISSUE THREAD

**Issue by christmart** [bigbluebutton/bigbluebutton#16427]
**Describe the bug**
In a long running connection (about 6 hours) to a BBB room a new webrtc connection is not possible.

**To Reproduce**
For instance, if another user shares his/her screen, all people who have joined longer than about 6 hours ago, will only see the circles indication a new screenshare is in progress. All other people are able to see the shared screen etc.
If the users with the problem exit the conference and reenter, they can see the screenshare.

**Expected behavior**
One should be able to get screenshares even if the conference connection runs longer than 6 hours.

**Actual behavior**
Screenshare stops working for long connected users after some hours.

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser: Firefox
 - Version: 102.6

**Comment by christmart** [comment]
BBB version at the moment is 2.5.8.
This is a problem that already exist for a long time (half a year or even more).
I think it came at least with the release of 2.5.

**Comment by Subdarub** [comment]
I am almost confident, that issue has existed for a quite a while in one form or another.
My colleagues and I basically have a daily conference, starting at the beginning of the work day lasting until the end of the work day. 

If I had to guess it became really prevelant with 2.3? Never the less the reason for this should be a terminated session. Because not only is the sceenshare not working you also wouldnt be able to join breakout rooms or turn on your webcam. 

My best guess is it might be related to some form of inactivity. I never bothered to make a real deep dive into this issue because it only happens in our "daily" conference. This issue has never been reported by a user.

**Comment by danimo** [comment]
Yes, it's been there for ages. It's still a problem for long workshop sessions though (which is not rare at all), so it should be fixed.

One observation: Hitting F5 when this happens (recalling this from memory) indicates that your session has expired.

**Comment by prlanzarin** [comment]
See https://github.com/bigbluebutton/bigbluebutton/pull/15741 which might help.
```
Java/Grails' HTTP session inactivity timeout is hardcoded to 4 hours.
This gives less flexibility for long-lived sessions - ie in rare
scenarios where users stay connected for >= that amount of time without
any re-connections or intermediate calls to checkAuthorization.

Ideally, we should look into a more permanent solution to avoid
invalidating perfectly healthy user sessions. This commit fixes
nothing in that regard.
```

**Comment by christmart** [comment]
This issue is still present in BBB 2.6 and 2.7.
